internal/cloudfront/origin.go

```diff
@@ -186,7 +186,9 @@ func (b OriginBuilder) addCachePolicyBehaviors(origin Origin) Origin {
 }
 
 func (b OriginBuilder) addBucketOriginConfiguration(origin Origin) Origin {
-	if b.originType != OriginTypeBucket {
+	origin.Type = b.originType
```

Comment:
Nice catch! nit: move it [here](https://github.com/gympass/cdn-origin-controller/pull/82/files/e030556720d0913bd41c65758251324326f07313..d11b132b24474b70d8d47864827dd8f91f8e4f84#diff-d4b20e40d720c2521ea434dadc743efcb453bc5ecd1d848bb98bab28ad3da175R141-R144), since this is not an optional field and we always want it to be set. I think it's a bit more explicit that way, wdyt?

Reply:
The commit for your "here" vanished with the rebase, but I assume you meant [here] 😉(https://github.com/gympass/cdn-origin-controller/blob/f1a71f84cfd6a7c7d88e40d36bd022c47198074a/internal/cloudfront/origin.go#L155)

Reply:
Ah, it was [here](https://github.com/gympass/cdn-origin-controller/blob/f1a71f84cfd6a7c7d88e40d36bd022c47198074a/internal/cloudfront/origin.go#L141-L144), but close enough hahaha